[{"id":828292663,"node_id":"R_kgDOMV6-Nw","name":"CVE-2024-34102-Python","full_name":"bughuntar/CVE-2024-34102-Python","private":false,"owner":{"login":"bughuntar","id":99727538,"node_id":"U_kgDOBfG4sg","avatar_url":"https://avatars.githubusercontent.com/u/99727538?v=4","url":"https://api.github.com/users/bughuntar","html_url":"https://github.com/bughuntar","type":"User","site_admin":false},"html_url":"https://github.com/bughuntar/CVE-2024-34102-Python","description":"CVE-2024-34102 Exploiter based on Python","fork":false,"url":"https://api.github.com/repos/bughuntar/CVE-2024-34102-Python","created_at":"2024-07-13T17:31:10Z","updated_at":"2024-07-13T18:17:27Z","pushed_at":"2024-07-13T18:16:18Z","stargazers_count":0,"watchers_count":0,"forks_count":0,"archived":false,"disabled":false,"allow_forking":true,"is_template":false,"topics":["cve-2024-34102"],"visibility":"public","forks":0,"watchers":0,"default_branch":"main","score":1},{"id":827743049,"node_id":"R_kgDOMVZbSQ","name":"CVE-2017-1235_exploit","full_name":"11k4r/CVE-2017-1235_exploit","private":false,"owner":{"login":"11k4r","id":81901900,"node_id":"MDQ6VXNlcjgxOTAxOTAw","avatar_url":"https://avatars.githubusercontent.com/u/81901900?v=4","url":"https://api.github.com/users/11k4r","html_url":"https://github.com/11k4r","type":"User","site_admin":false},"html_url":"https://github.com/11k4r/CVE-2017-1235_exploit","description":"","fork":false,"url":"https://api.github.com/repos/11k4r/CVE-2017-1235_exploit","created_at":"2024-07-12T09:25:31Z","updated_at":"2024-07-13T18:12:32Z","pushed_at":"2024-07-13T18:12:28Z","stargazers_count":0,"watchers_count":0,"forks_count":0,"archived":false,"disabled":false,"allow_forking":true,"is_template":false,"topics":[],"visibility":"public","forks":0,"watchers":0,"default_branch":"main","score":1}]